openhtf/core/test_state.py

```diff
@@ -68,6 +68,26 @@ class DuplicateAttachmentError(Exception):
   """Raised when two attachments are attached with the same name."""
 
 
+class Subscribable(object):
```

Comment:
I like the idea of separating this out, but it might go better in the `util` submodule.